### ISSUE TITLE
Python install fails on Windows

### ISSUE THREAD

**Issue by spigo900** [NextCenturyCorporation/AIDA-Interchange-Format#255]
Installing the Python package fails on Windows because the `where` path passed to `setuptools.find_packages()` ends in a slash:

```
> pip install .
Processing e:\shared\coding\isi\2019-aida\repos\aida-interchange-format
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\crazy\AppData\Local\Temp\pip-req-build-91ixkqom\setup.py", line 18, in <module>
        packages=setuptools.find_packages(where='./python/', exclude=['tests']),
      File "c:\users\crazy\appdata\local\programs\python\python37\lib\site-packages\setuptools\__init__.py", line 58, in find
        convert_path(where),
      File "c:\users\crazy\appdata\local\programs\python\python37\lib\distutils\util.py", line 112, in convert_path
        raise ValueError("path '%s' cannot end with '/'" % pathname)
    ValueError: path './python/' cannot end with '/'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\crazy\AppData\Local\Temp\pip-req-build-91ixkqom\
```

**Comment by craigwarsaw** [comment]
Thanks for submitting this bug report.  Is this a show-stopping issue for you or do you have a work-around?

**Comment by spigo900** [comment]
Not show-stopping, no. To work around it, all I had to do was to get rid of the `/` at the end of the path.

**Comment by craigwarsaw** [comment]
Thanks - we'll get it updated in our next release (to be scheduled).

**Comment by nextcen-dgemoets** [comment]
This is fixed in the recently-released AIF v1.0.4.  Closing this issue.
